Classes/Controller/AbstractController.php

```diff
@@ -566,15 +602,19 @@ protected function buildSimplePagination(PaginationInterface $pagination, Pagina
      */
     private function getDocumentByUid(int $documentId)
     {
-        // TODO: implement multiView as it is in getDocumentByUrl
         $doc = null;
         $this->document = $this->documentRepository->findOneByIdAndSettings($documentId);
 
         if ($this->document) {
-            $doc = AbstractDocument::getInstance($this->document->getLocation(), $this->settings, false);
-            // fix for count(): Argument #1 ($value) must be of type Countable|array, null given
-            $this->documentArray[] = $doc;
-        } else {
+            $doc = AbstractDocument::getInstance($this->document->getLocation(), $this->settings);
+            if ($doc !== null) {
+                $this->buildMultiView($doc);
+                // fix for count(): Argument #1 ($value) must be of type Countable|array, null given
+                $this->documentArray[] = $doc;
+            }
+        }
+
+        if ($this->document || $doc === null) {
```

Comment:
The normal good case (valid `$this->document`) now logs an error.

Reply:
If there is an issue, please create a issue with the error message and the call to reproduce it, if necessary, including a reference to this change. Otherwise, the topic might get lost in the merged and closed pull request.

Reply:
Pull request #1554 addresses this issue.